gammapy/spectrum/tests/test_counts_spectrum.py

```diff
@@ -1,10 +1,17 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
```

Comment:
Let's keep this on one line.
We have > 100 of those lines in Gammapy, and I put them all on one line.